src/index.css

```diff
@@ -450,3 +450,11 @@ div.monokleEditorErrorRefGlyphClass.monokleEditorUnsatisfiedRefGlyphClass::after
 .terminal-options-menu .ant-dropdown-menu-item:hover {
   background-color: transparent;
 }
+
+.walkthrough-modal ul li::marker {
+  color: #1d1d1d;
+}
+
+.walkthrough-modal .ant-modal-content .ant-modal-body {
```

Comment:
Is this something that can be fixed inside the core component also? 🤔

Reply:
We cannot fix it from there because the Monokle desktop css is overriding the modal background color